### PR TITLE
Update /firefox/new/quantum.lang

### PR DESCRIPTION
**Pull request by alexgibson**
New strings added behind the tag `platform_strings_04232019` for https://github.com/mozilla/bedrock/pull/7098

### CODE REVIEW

**Comment by peiying2** [comment]
@alexgibson I didn't spot error in the new strings added, but there are two main issues to be fixed and it takes a bit of work:
* Add the new tag on top of the file in the note section. Try to place it in alphabetic/numerical order.
* Bind the newly tagged strings with the old strings so they are next to each other (like in the template PR). 

These sections provide details on how to update an existing file:
* https://github.com/mozilla-l10n/documentation/blob/master/products/mozilla_org/working_bedrock.md#l10n-tags
* https://github.com/mozilla-l10n/documentation/blob/master/products/mozilla_org/extract_strings.md#add-notes-and-comments-to-the-extracted-file. Click on the links for .lang examples.

**Comment by alexgibson** [comment]
@peiying2 thanks. I added the tag and reordered the best I could.

File: en-US/firefox/new/quantum.lang [hunk 1]
```diff
@@ -222,39 +264,168 @@ Download Firefox
...
+
+
+## TAG: platform_strings_04232019
+# /firefox/linux/ HTML page description
+;Download Mozilla Firefox for Linux, a free Web browser. Firefox is created by a global non-profit dedicated to putting individuals in control online. Get Firefox for Linux today!
+Download Mozilla Firefox for Linux, a free Web browser. Firefox is created by a global non-profit dedicated to putting individuals in control online. Get Firefox for Linux today!
+
+
 # HTML page description
 ;Download Mozilla Firefox, a free Web browser. Firefox is created by a global non-profit dedicated to putting individuals in control online. Get Firefox for Windows, macOS, Linux, Android and iOS today!
```
Comment (peiying2) [inline en-US/firefox/new/quantum.lang]: The obsolete strings will be marked with a comment: `# The string is obsolete. Do note remove.` Please find the pairs and add the comment.

**Comment by peiying2** [comment]
@alexgibson the replaced strings should include a comment to indicate they are obsolete https://github.com/mozilla-l10n/www.mozilla.org/pull/294/files#r278805986. This file contains a few of those examples. They are usually next to the newly tagged strings you just reordered.  After this update, I will take a look again. It looks good otherwise.

**Comment by alexgibson** [comment]
@peiying2 the strings are not obsolete, they are still in use on /firefox/new.

File: en-US/firefox/new/quantum.lang (hunk 1 above)
Comment (alexgibson) [reply en-US/firefox/new/quantum.lang]: These strings are not obsolete, so shouldn’t be marked as such. They are still in use on /firefox/new/.

**Comment by peiying2** [comment]
> Looking at https://l10n.mozilla-community.org/langchecker/?locale=all&website=0&file=firefox/new/quantum.lang, I have a hard time reasoning about which obsolete strings can be safely removed. Please provide some guidance if you would like additional cleanup here? Thanks

Before you add the "obsolete" comments, you can check the existing ones, and a few no longer exist in the pages unless they are used else where. 
- `Meet Firefox Quantum`, `New. Fast. For good.`, `Download in another language`

**Comment by peiying2** [comment]
> @peiying2 the newly replaced strings are not obsolete, they are still in use on /firefox/new.
> 
This is to indicate that the string is an older version. Still in use if the newly tagged is not localized. To make it go away, it needs to be physically removed from the file. Here it is just a comment.

> How do I tell which currently obsolete strings can be removed?

This is a manual process, unfortunately.

**Pull request merged**